### ISSUE TITLE
Add test coverage for cypress

### ISSUE THREAD

**Issue by kushthedude** [OpenRefine/OpenRefine#3461]
We are not tracking coverage for ui_tests.
### Proposed solution

It would be great if we can capture the coverage outputs while testing UI too.

**Comment by wetneb** [comment]
Are you thinking about frontend code coverage? @fgiroud mentioned that we could potentially also track the backend coverage of these tests, since they really are integration tests (as we are not mocking the backend).

**Comment by kushthedude** [comment]
> Are you thinking about frontend code coverage? @fgiroud mentioned that we could potentially also track the backend coverage of these tests, since they really are integration tests (as we are not mocking the backend).

Yes, I was initially thinking about frontend code coverage only.
I am unsure about how would we track backend coverage, any resources highlighting the same can be helpful and would help in further investigation.

**Comment by fgiroud** [comment]
Frontend code coverage is theoretically possible, and it would require to instrument all the frontend javascript code used.
[This article is a good start](https://docs.cypress.io/guides/tooling/code-coverage.html#Introduction)
